presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -1484,6 +1484,36 @@ public void testComparison()
         assertFunction("ARRAY [ARRAY [1, 2], ARRAY [3, 4, 5]] < ARRAY [ARRAY [1, 2], ARRAY [3, 4, 5]]", BOOLEAN, false);
     }
 
+    @Test
+    public void testArrayContainsArray()
```

Comment:
This is about 20 separate tests wrapped into one method. Splitting them out into 20 test methods makes them unit tests that can pass or fail and be debugged independently. As is a single failure hides subsequent failures.

Reply:
Not sure if I followed it. @elharo are you recommending to split all the test into individual method?

Reply:
No, absolutely not. I feel like I need to write some articles about this because devs keep making the same bad arguments (and by no means just in this project) instead of simply following long-established best practices for unit tests. To be explicit:

1. One test per one test method. Independent tests should pass or fail independently.
2. Use fixtures for common state (though you don't even need that here).
3. Use more test classes. Don't put all tests in one test class.
4. Most importantly, no cargo cult programming. Do what's right even if existing code does it wrong. Consistency with bad code is not a virtue.

Reply:
@elharo Created separate test cases for individual test type. Let me know if it looks good.